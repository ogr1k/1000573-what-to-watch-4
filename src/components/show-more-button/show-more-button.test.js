import React from "react";
import renderer from "react-test-renderer";
import ShowMoreButton from "./show-more-button.jsx";

it(`Render ShowMoreButton`, () => {
  const tree = renderer
    .create(<ShowMoreButton onCLick={() => {}}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
