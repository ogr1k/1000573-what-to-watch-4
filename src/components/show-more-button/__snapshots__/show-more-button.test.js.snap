// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ShowMoreButton 1`] = `
<div
  className="catalog__more"
  onClick={[Function]}
>
  <button
    className="catalog__button"
    type="button"
  >
    Show more
  </button>
</div>
`;
